### ISSUE TITLE
Delete device icon name not accessible

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4098]
## Area or Component
Facility form

## Quick Summary
The trash can icon to remove a device is labelled simply "delete device". However, if there are multiple devices for the facility, this name is not unique. It also doesn't tell users which device they're about to delete.

## Accessibility Violation
Name: The element's name is missing or incorrect: https://cdc-devtools.dequecloud.com/rule-help/en/aria-name-missing-incorrect

## Action Requested
Rename the aria-label on the trash can icon to be more specific and include the selected device's name. "Delete Abbot BinaxNow" is much more descriptive than simply "delete device".

## Screenshots (optional)
<img width="950" alt="Screen Shot 2022-07-27 at 4 29 32 PM" src="https://user-images.githubusercontent.com/80282552/181389694-72a5f9b8-03d0-4e72-b5b1-1527a005bb93.png">

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=1346058064&range=12:13

**Comment by georgehager** [comment]
This ticket should be resolved when #4269 is resolved by PR #4314.
